modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateSupportingDocuments < ActiveRecord::Migration
```

Comment:
I thought protocol requires us to put this in a separate PR?

Reply:
it is, I'll split it out once this pr is approved